include/swift/Basic/Timer.h

```diff
@@ -45,6 +45,38 @@ namespace swift {
       CompilationTimersEnabled = State::Enabled;
     }
   };
+  
+  // A SharedTimer for recursive routines.
```

Comment:
This should be 3 slashes for it to be a doxygen comment ala LLVM style:

https://llvm.org/docs/CodingStandards.html#doxygen-use-in-documentation-comments